### PR TITLE
COR-48 Update packages + audit fix

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**
- Update Typescript for all packages + fix new compiler errors
- Update Sanity packages
- Update styled-components + styled-system
- Audit fix
- Use `tsconfig-replace-paths` instead of `tscpaths`

### CODE REVIEW

File: packages/common/package.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.27.1",
     "prettier": "^2.5.0",
-    "ts-node": "^10.4.0",
+    "ts-node": "^10.8.1",
     "tsconfig-paths": "^3.12.0",
-    "tscpaths": "^0.0.9",
-    "typescript": "^4.4.4",
+    "tsconfig-replace-paths": "^0.0.11",
```
Comment (VWSCoronaDashboard24) [inline packages/common/package.json:43]: The `tscpaths` package was abandoned 4 years ago. `tsconfig-replace-paths` is a fork that is kept up-to-date.

**Comment by VWSCoronaDashboard19** [comment]
Is this PR related to: https://github.com/minvws/nl-covid19-data-dashboard/pull/4248 ?

File: packages/cms/src/data/data-structure.ts [hunk 2]
```diff
@@ -27,22 +27,32 @@ export const dataStructure = {
     vaccine_coverage_per_age_group: [
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard24) [reply packages/cms/src/data/data-structure.ts]: Fixed

**Comment by VWSCoronaDashboard24** [comment]
> Is this PR related to: #4248 ?

Yep, the audit fix is also part of this PR. But I did it myself in this PR.

**Review by VWSCoronaDashboard19 — APPROVED** [review]

**Pull request merged**